docs/data/parquet_files.md

```diff
@@ -0,0 +1,87 @@
+# Parquet files you might encounter throughout PIXL
+
+## OMOP-ES files
+
+From
+[OMOP-ES](https://github.com/UCLH-Foundry/the-rolling-skeleton/blob/main/docs/design/100-day-design.md#data-flow-through-components)
+we receive parquet files defining the data we need to export. These input files appear as 2 groups:
+
+1. **Public** parquet files: have had identifiers removed and replaced with a sequential ID for the
+   export
+2. **Private** parquet files: map sequential identifiers to patient identifiers (e.g. MRNs,
+   Accession numbers, NHS numbers)
+
+## Radiology reports
+
+In addition to the OMOP-ES files, the PIXL pipeline generates **Radiology** parquet files, which
+contain the radiology reports for the given extract. These are generated by collecting all
+_de-identified_ radiology reports for the given extract from the CogStack API.
+
+The functionality for this is defined in the [EHR API](../../pixl_ehr/README.md), specifically in
+[`PIXLDatabase.get_radiology_reports`](../../pixl_ehr/src/pixl_ehr/_databases.py), which queries the
+<!-- TODO: add link to PIXL database documentation once we have that -->
+[PIXL database]() for the de-identified radiology reports of the current extract and collects them
+in a single _parquet_ file together with the `image_identifier` and `procedure_occurrence_id`.
+
+_This was implemented in [PR#203](https://github.com/UCLH-Foundry/PIXL/pull/203)_.
```

Comment:
Feels a bit OTT to document which PR implemented it but not against it, may go stale if we update the code is my only concern